devtools/packaging/scripts/osx/build.sh

```diff
@@ -37,7 +38,7 @@ fi
 mkdir build
 cd build
 cmake ../openmm $CMAKE_FLAGS
-make -j4 all DoxygenApiDocs sphinxpdf
+make -j4 all PythonInstall C++ApiDocs PythonApiDocs sphinxpdf
```

Comment:
what a comedy of errors. you need to run `make install` before you can run `make PythonInstall`